pycbc/events/coinc.py

```diff
@@ -1170,6 +1175,9 @@ def _find_coincs(self, results, valid_ifos):
                 trig_stat = trigs['stat'][i]
                 trig_time = trigs['end_time'][i]
                 template = trigs['template_id'][i]
+                mass1 = trigs['mass1'][i]
+                mass2 = trigs['mass2'][i]
+                mchirp = mchirp_from_mass1_mass2(mass1, mass2)
```

Comment:
Doing this conversion one trigger at a time may be very very slow, maybe do them all using array operations first?

Reply:
I've made this change but I need to update the `test_live_coinc_compare.py` test case to include both 'mass1' and 'mass2' in the trigs dict. They are present in Live (need to quadruple check) but not in the test case.

Reply:
Resolved?